### DISCUSSION TITLE
Report a net:: ERR_ABORTED 404 error

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#86] [Q&A] [answered]
I deployed a set of lotus docs themes, but there is always a resource that cannot be found.
Although I haven't found any problem yet, I just want to ask what may be the cause?

This is the [docs link](https://rapidai.github.io/RapidOCRDocs/docs/business_support/). You can press **F12**, and click **Console** page. An error message similar to the following will appear:
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/37434e59-4d66-40d9-9f1d-7d53599b9939)

Appreciate any help on this.

**Comment by colinwilson** [comment, marked as answer]
If you haven't got Plausible Analytics configured, comment out the [plausibe config](https://github.com/RapidAI/RapidOCRDocs/blob/595033344fbb0c5bf832f8a408b98c665f77e693/hugo.toml#L108-L110) in your `hugo.toml` config.
